hbase_master/README.md

```diff
@@ -67,7 +92,7 @@ The Hbase_master check does not include any service checks.
 
 ## Troubleshooting
 
-Need help? Contact [Datadog support][12].
+Need help? Contact [Datadog support][14].
```

Comment:
```suggestion
Need help? Contact [Datadog support][12].
```

Reply:
And this is how I finally understood how reference-style links work